packages/components/src/Popover/_docs/Popover.stories.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react"
+import { Meta, StoryObj } from "@storybook/react"
+import { AddIcon, MeatballsIcon } from "~components/Icon"
+import { Popover as PopoverComponent, PopoverProps, usePopover } from "../index"
+
+const meta = {
+  title: "Components/Popover",
+  component: PopoverComponent,
+  args: {
+    children: undefined,
+    referenceElement: undefined,
+  },
+  argTypes: {
+    customIcon: {
+      options: ["None", "MeatballsIcon", "AddIcon"],
+      control: { type: "radio" },
+      mapping: {
+        None: undefined,
+        MeatballsIcon: <MeatballsIcon role="presentation" />,
+        AddIcon: <AddIcon role="presentation" />,
+      },
+    },
+  },
+} satisfies Meta<typeof PopoverComponent>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+const PopoverWrapper = (
+  args: Omit<PopoverProps, "children" | "referenceElement">
+): JSX.Element => {
+  const [referenceElementRef, Popover] = usePopover()
+
+  return (
+    <div className="text-center relative">
+      <button
+        type="button"
+        className="inline-block mt-112"
+        ref={referenceElementRef}
+      >
+        Pop
+      </button>
+      <Popover {...args}>
+        Popover body that explains something useful.{" "}
+        <a href="/">Optional link</a>
+      </Popover>
+    </div>
+  )
+}
+
+export const Playground: Story = {
```

Comment:
Have source default open in docs?

Reply:
Intentional, cause the source pushes the controls so far down that it's hard to use.